code/datums/components/riding/riding_mob.dm

```diff
@@ -521,3 +532,113 @@
 /datum/component/riding/leaper/handle_unbuckle(mob/living/rider)
 	. = ..()
 	UnregisterSignal(rider,  COMSIG_MOB_POINTED)
+
+/datum/component/riding/creature/raptor
+	require_minigame = TRUE
+
+/datum/component/riding/creature/raptor/handle_specials()
+	. = ..()
+	set_riding_offsets(RIDING_OFFSET_ALL, list(TEXT_NORTH = list(0, 8), TEXT_SOUTH = list(0, 8), TEXT_EAST = list(-4, 8), TEXT_WEST = list(4, 8)))
+	set_vehicle_dir_layer(SOUTH, ABOVE_MOB_LAYER)
+	set_vehicle_dir_layer(NORTH, OBJ_LAYER)
+	set_vehicle_dir_layer(EAST, OBJ_LAYER)
+	set_vehicle_dir_layer(WEST, OBJ_LAYER)
+
+/datum/component/riding/creature/raptor/fast
+	vehicle_move_delay = 1.5
+
+/datum/riding_minigame
```

Comment:
Also a class level documentation for this would be nice. What it's used for, etc.